cosmwasm/ibc-union/lightclient/ethereum/src/errors.rs

```diff
@@ -55,6 +55,12 @@ pub enum Error {
 
     #[error("the misbehaviour headers must be different")]
     IdenticalMisbehaviourHeaders,
+
+    #[error("aggregate pubkey mismatch")]
+    AggregatePubkeyMismatch,
```

Comment:
why not include the keys? or will this error be removed soon

Reply:
it's gonna be removed like this week